dace/codegen/codeobject.py

```diff
@@ -53,6 +54,9 @@ def __init__(self,
         self.linkable = linkable
         self.environments = environments or set()
 
+        if (language == "cpp") & (title == "Frame"):
```

Comment:
Not standard python. Should be `if language == 'cpp' and title == 'Frame':`

Also see coding guidelines at [CONTRIBUTING.md](https://github.com/spcl/dace/blob/master/CONTRIBUTING.md)